lib/hbs-parser.js

```diff
@@ -1,11 +1,18 @@
 const createHbsGulpStream = (partials, dataObject, dataGlob, debug=false) => {
-	// @TODO add possibility to inject helpers from external file
-	const hbsHelpers = require('./hbs-helpers');
 	const hb = require('gulp-hb');
+	const hbsHelpers = require('./hbs-helpers');
+	const cwd = process.cwd();
+	const config = require('./../config');
+	const projectHbsHelpersPath = '/' + config.global.src + config.global.resources + config.global.handlebarsHelper;
```

Comment:
I would suggest to use the path module to join paths like that for all newly created paths. Best case would be to use full path with cwd.
So this should be something like path.join(cwd, config.global.src, config.global.resources, config.global.handlebarsHelper)